src/main/java/io/vavr/control/Option.java

```diff
@@ -429,6 +445,19 @@ public final T getOrElse(T other) {
     /**
      * Returns this {@code Option} if it is nonempty, otherwise return the alternative.
      *
+     * <pre>{@code
+     * Option<String> other = Option.of("Other");
+     *
+     * // Creates Some("Hello World")
+     * Option.of("Hello World").orElse(other);
+     *
+     * // Creates Some("Other")
+     * Option.none().orElse(other);
+     *
+     * // Creates None
+     * Option.none().orElse(Option.none())
```

Comment:
I think the goal of this example is a bit confusing... From the comment _"Creates None"_, it's not evident to know if the result comes from `Option.none()` or `.orElse(Option.none())`. I suggest removing this entry.